src/internal/m365/onedrive/permission.go

```diff
@@ -240,13 +291,13 @@ func UpdateLinkShares(
 	itemID string,
 	lsAdded, lsRemoved []metadata.LinkShare,
 	oldLinkShareIDToNewID map[string]string,
-) error {
-	// You can only delete inherited sharing links the first time you
+) (bool, error) {
+	// You can only delete inherited sharing links the didReset time you
```

Comment:
So is `didReset` the second time, or the seventh?

Reply:
Find and replace fails 😅